components/list/list.js

```diff
@@ -136,8 +137,8 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 			<slot name="header"></slot>
 			<div role="${role}" aria-label="${ifDefined(ariaLabel)}" class="d2l-list-content">
 				<slot @keydown="${this._handleKeyDown}" @slotchange="${this._handleSlotChange}"></slot>
+				${this._renderPagerContainer()}
```

Comment:
I think we previously moved non-list-item elements out of the the `role="list"` element to avoid incorrect screen reader announcement for list-item counts.

Reply:
Yeah I think that is true. I'm not super keen on changing where the pager renders based on the role though. I feel like there should be another way to address the issue (which sounds really strange in the description).